core/server/data/migrations/versions/2.3/1-add-webhook-columns.js

```diff
@@ -0,0 +1,38 @@
+const common = require('../../../../lib/common');
+const commands = require('../../../schema').commands;
+const table = 'webhooks';
+const newColumnNames = ['name', 'integration_id', 'secret', 'last_triggered_at', 'api_version'];
```

Comment:
Should we also add a field to track the result of the trigger?
Even if we just add it later.

`last_triggered_at: X`
`status: successful|failed with reason`

Reply:
Yeah, the UI has an area to display the status and a failure message so being able to show it in the near future (maybe not ready for end of this cycle) would be useful